src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,116 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private int stSize = 10;
```

Comment:
It should be constant. [Follow the name convention ](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)